great_expectations/data_context/data_context.py

```diff
@@ -1677,14 +1667,19 @@ def add_datasource(self, name, type_, **kwargs):
 
 class ExplorerDataContext(ConfigOnlyDataContext):
 
-    def __init__(self, config, expectation_explorer=False, data_asset_name_delimiter='/'):
+    def __init__(self, config, context_root_dir=None, expectation_explorer=True, data_asset_name_delimiter='/'):
```

Comment:
Good catch.

An alternative would be to subclass ExplorerDataContext directly from DataContext, instead of ConfigOnlyDataContext...